.nuspec/Microsoft.Maui.Controls.SingleProject.props

```
@@ -6,7 +6,7 @@
     <WindowsAppSDKSelfContained Condition=" '$(WindowsAppSDKSelfContained)' == '' and '$(WindowsPackageType)' == 'None' and '$(OutputType)' == 'WinExe' ">true</WindowsAppSDKSelfContained>
     <WindowsAppSdkBootstrapInitialize Condition=" '$(WindowsAppSdkBootstrapInitialize)' == '' and '$(EnableMsixTooling)' == 'true' and '$(OutputType)' != 'WinExe' ">false</WindowsAppSdkBootstrapInitialize>
     <PublishAppXPackage Condition=" '$(PublishAppXPackage)' == '' and '$(EnableMsixTooling)' == 'true' and '$(WindowsPackageType)' == 'MSIX' ">true</PublishAppXPackage>
-    <PublishReadyToRun Condition=" '$(Configuration)' == 'Release' and '$(PublishReadyToRun)' == '' ">true</PublishReadyToRun>
+    <PublishReadyToRun Condition=" '$(PublishReadyToRun)' == '' and '$(Configuration)' == 'Release' and '$(OutputType)' == 'WinExe' ">true</PublishReadyToRun>
```

Can we do:

```suggestion
    <PublishReadyToRun Condition=" '$(PublishReadyToRun)' == '' and '$(Configuration)' == 'Release' and '$(OutputType)' != 'Library' ">true</PublishReadyToRun>
```

`ReadyToRun` should be fine for console apps?

---

This is just for maui and for libraries it is also ''. So if you do a File | New | Class Library, there is no output type set and this runs too early I think I saw.

WinExe is fine as this runs just after the line above:

```xml
<OutputType Condition="'$(OutputType)' == 'Exe'">WinExe</OutputType>
```

This runs as the windows TFM, and we _always_ set the output type to WinExe